src/app.js

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2015 Colin Clark
+
+Licensed under the 3-Clause "New" BSD license.
+You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the 3-Clause "New" BSD License at
+https://github.com/colinbdclark/infusion-electron/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    electron = fluid.registerNamespace("electron");
+
+// Monkey-patch fluid.log for Electron's broken console.log.
```

Comment:
https://github.com/atom/electron/issues/1368

I'll update the comment to describe the issue and link to their bug tracker.